src/contributing/structure.md

```diff
@@ -0,0 +1,21 @@
+# Documentation Structure
+
+The directory structure is meant to feel comfortable for those familiar with the data platform:
+
+```
+.
+|--src
+   |--datasets - contains dataset level documentation
+   |--tools - contains tool level documentation
+   |--concepts - contains tutorials meant to introduce a new concept to the reader
+   |--cookbooks - focused code examples for reference
+```
+
+This documentation is meant to take the reader from beginner to expert.
```

Comment:
This documentation takes you from being a beginner to being an expert.

Reply:
I think 'meant' is a better way of phrasing this, since it's not a foregone conclusion that someone will go from beginner to expert.